TFT/src/User/Menu/ABL.c

```diff
@@ -66,8 +67,7 @@ void ablStart(void)
 
     case BL_UBL:  // if Unified Bed Leveling
       storeCmd("G29 P1\n");
-      // Run this multiple times since it only fills some missing points, not all.
-      storeCmd("G29 P3\n");
+      storeCmd("G29 P3\n");  // run this multiple times since it only fills some missing points, not all
```

Comment:
Here the fist alphabet is changed to small?

Reply:
See my first comment. Just used same form in the whole file